### PR TITLE
Fix managed nuget to have ref assembly marked AnyCpu

### PR DESCRIPTION
**Pull request by dannyvv**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6493)

### CODE REVIEW

**Comment by dannyvv** [comment]
This is a blocker for the TODO app, hence the request to backport to 0.63.

File: vnext/Scripts/Tfs/Layout-Headers.ps1 [hunk 1]
```diff
@@ -120,3 +120,34 @@ Copy-Item -Force -Path $ReactWindowsRoot\Scripts\Microsoft.ReactNative.Managed.C
...
+
+# There unfortunately is no flag to csc, and no single tool to do this. CorFlags.exe does not have support to change this...
+# This is a lot more straight-forward than trying to create an AnyCpu version of this assembly, update the solution, tests and add
+# another platform to the CI matrix, not to mention the steps needed to ensure AnyCpu ran for the x64 build to test the nuget package
+# in PR validation....
+
+# Therefore this step will simply ildasm and ilasm the reference assembly and store it as which strips the bitness and generates anycpu msil.
+# These reference assemblies are just regular assemblies but with empty bodies
+
+ForEach ($refFolder in (Get-ChildItem -Path "d:\src\rnw1\vnext\target" -Recurse -Include "ref"))
```
Comment (asklar) [inline vnext/Scripts/Tfs/Layout-Headers.ps1:139]: hardcoded path?

**Review by asklar — CHANGES_REQUESTED** [review]
Making sure automerge doesn't merge the hardcored path :)

File: vnext/Scripts/Tfs/Layout-Headers.ps1 (hunk 1 above)
Comment (dannyvv) [reply vnext/Scripts/Tfs/Layout-Headers.ps1:139]: oops, was for testing...

**Review by asklar — APPROVED** [review]

**Comment by dannyvv** [comment]
Note: I cross-posted PR's between #6474 and #6493
Renaming the PR's and comments ....

**Pull request merged**